code/modules/admin/IsBanned.dm

```diff
@@ -109,91 +109,79 @@
-				else
-					log_admin("The admin [key] has been allowed to bypass a matching ban on [pckey]")
-					message_admins("<span class='adminnotice'>The admin [key] has been allowed to bypass a matching ban on [pckey]</span>")
-					addclientmessage(ckey,"<span class='adminnotice'>You have been allowed to bypass a matching ban on [pckey].</span>")
-					continue
-			var/expires = ""
-			if(text2num(duration) > 0)
-				expires = " The ban is for [duration] minutes and expires on [expiration] (server time)."
+				log_admin("The admin [key] is admin banned, and has been disallowed access")
+				message_admins("<span class='adminnotice'>The admin [key] is admin banned, and has been disallowed access</span>")
 			else
-				var/appealmessage = ""
-				if(GLOB.configuration.url.banappeals_url)
-					appealmessage = " You may appeal it at <a href='[GLOB.configuration.url.banappeals_url]'>[GLOB.configuration.url.banappeals_url]</a>."
-				expires = " This ban does not expire automatically and must be appealed.[appealmessage]"
+				log_admin("The admin [key] has been allowed to bypass a matching ban on [pckey]")
+				message_admins("<span class='adminnotice'>The admin [key] has been allowed to bypass a matching ban on [pckey]</span>")
+				addclientmessage(ckey,"<span class='adminnotice'>You have been allowed to bypass a matching ban on [pckey].</span>")
+				continue
+		var/expires = ""
+		if(text2num(duration) > 0)
+			expires = " The ban is for [duration] minutes and expires on [expiration] (server time)."
+		else
+			var/appealmessage = ""
+			if(GLOB.configuration.url.banappeals_url)
+				appealmessage = " You may appeal it at <a href='[GLOB.configuration.url.banappeals_url]'>[GLOB.configuration.url.banappeals_url]</a>."
+			expires = " This ban does not expire automatically and must be appealed.[appealmessage]"
 
-			var/desc = "\nReason: You, or another user of this computer or connection ([pckey]) is banned from playing here. The ban reason is:\n[reason]\nThis ban was applied by [ackey] on [bantime][ban_round_id ? " (Round [ban_round_id])" : ""].[expires]"
+		var/desc = "\nReason: You, or another user of this computer or connection ([pckey]) is banned from playing here. The ban reason is:\n[reason]\nThis ban was applied by [ackey] on [bantime][ban_round_id ? " (Round [ban_round_id])" : ""].[expires]"
 
-			. = list("reason"="[bantype]", "desc"="[desc]")
+		. = list("reason"="[bantype]", "desc"="[desc]")
 
-			log_adminwarn("Failed Login: [key] [computer_id] [address] - Banned [.["reason"]]")
-			INVOKE_ASYNC(GLOBAL_PROC, .proc/log_connection, ckey(key), address, computer_id, CONNECTION_TYPE_DROPPED_BANNED)
-			qdel(query)
-			return .
+		log_adminwarn("Failed Login: [key] [computer_id] [address] - Banned [.["reason"]]")
+		INVOKE_ASYNC(GLOBAL_PROC, .proc/log_connection, ckey(key), address, computer_id, CONNECTION_TYPE_DROPPED_BANNED)
 		qdel(query)
+		return .
+	qdel(query)
 
 	. = ..()	//default pager ban stuff
 	if(.)
```

Comment:
Part of me feels like this needs moving up but the comment below makes me not think that
```dm
		//byond will not trigger isbanned() for "global" host bans,
		//ie, ones where the "apply to this game only" checkbox is not checked (defaults to not checked)
		//So it's safe to let admins walk thru host/sticky bans here
```